courses/olomouc.yml

```diff
@@ -0,0 +1,318 @@
+    
+    Stránky samotných PyLadies najdeš na http://pyladies.cz.
+
+    Domácí úkoly k jednotlivým lekcím najdeš na https://projekty.pyladies.cz/.
+
+    Zpětnou vazbu na konci každé lekce můžeš zadat na https://zpetnovazebnik.pyladies.cz/. 
+vars:
+  pyladies: true
+  coach-present: true
+sessions:
+- slug: preparation
+  serial: 0
+  title: Příprava před začátkem kurzu
+  materials:
+  - title: Studijní materiály
+  - lesson: beginners/cmdline
+  - lesson: beginners/install
+  - lesson: beginners/venv-setup
+  - lesson: beginners/install-editor
+  - title: Videa ke shlédnutí
+  - title: (13:50) Příkazová řádka - Linux & macOS - video
+    url: https://youtu.be/GGMg8O4hE0c 
+  - title: (22:34) Příkazová řádka - Windows - video
+    url: https://youtu.be/kriVWJmXpZc
+  - title: (8:15) Instalace Pythonu - Linux - video
+    url: https://youtu.be/7Qi7cSkoBA0
+  - title: (4:48) Instalace Pythonu - Windows - video
+    url: https://youtu.be/Xr6liKJzRGA
+  - title: (7:48) Nastavení prostředí - video
+    url: https://youtu.be/AFVvpfQB0V0
+  - title: (8:01) Instalace editoru - video
+    url: https://youtu.be/JcXUCuneX04
+  - title: Další odkazy a materiály
+  - title: Tahák na klávesnici (PDF)
+    url: https://pyvec.github.io/cheatsheets/keyboard/keyboard-cs.pdf
+    type: cheatsheet
+
+- slug: hello_world
+  title: První program
+  date: 2024-10-08
+  materials:
+  - title: Studijní materiály
+  - lesson: beginners/first-steps
+  - lesson: beginners/hello-world
+  - lesson: beginners/print
+  - lesson: beginners/variables
+  - title: Videa ke shlédnutí
+  - title: (7:23) První krůčky - video
+    url: https://youtu.be/0Poe25XdKIA
+  - title: (7:50) První program - video
+    url: https://youtu.be/9aJcvOfleFs
+  - title: (12:38) Print a chybové hlášky - video
+    url: https://youtu.be/14-LVG9Edng
+  - title: (15:08) Proměnné - video
+    url: https://youtu.be/-zKws24FmCg
+  - title: Další odkazy a materiály
+  - title: Tahák na debuggování (PDF)
+    url: https://pyvec.github.io/cheatsheets/errors/errors-cs.pdf
+    type: cheatsheet
+
+- slug: conditions
+  title: Podmínky a funkce
+  date: 2024-10-15
+  materials: 
+  - title: Studijní materiály
+  - lesson: beginners/expressions
+  - lesson: beginners/comparisons
+  - lesson: beginners/functions
+  - lesson: beginners/basic-functions
+  - lesson: beginners/and-or
+  - title: Videa ke shlédnutí
+  - title: (8:32) Vyhodnocování výrazů - video
+    url: https://youtu.be/nNXDXuN6Smg
+  - title: (17:27) Porovnání - video
+    url: https://youtu.be/Q1YJqWzfnck
+  - title: (17:26) Funkce - video
+    url: https://youtu.be/w_d8VKS8i48
+  - title: (15:51) Užitečné funkce - video
+    url: https://youtu.be/udEOvGIuZrs
+  - title: (7:58) Nebo anebo a - video
+    url: https://youtu.be/5Mc-cgoaM10
+  - title: Další odkazy a materiály
+  - title: Tahák na užitečné funkce (PDF)
+    url: https://pyvec.github.io/cheatsheets/basic-functions/basic-functions-cs.pdf
+    type: cheatsheet
+
+- slug: loops
```

Comment:
Přijde mi toho strašně moc, vypadá to děsivě IMO :laughing: Přimlouval bych se za to, abychom nechali pouze první lekci, případně abychom nechali kostru lekcí (aby šel vidět časový plán), ale dali tam TODO:
![image](https://github.com/user-attachments/assets/c1597e87-a8c8-4710-8d25-d1f4c9f971b2)

Reply:
Osobne, ak tam dáme, že to "bude doplneno" posobí to dosť neprofesionálne. Programovanie nie je jednoduchý koníček. Dalšia vec je, že keď budeš mať niekoho šikovného tak ho budeš iba brzdiť.

Reply:
Tak ony ty matroše se dají najít na jiných kurzech, je to pořád to stejné. A ono to stejně bude doplňováno dynamicky, nepamatuju, že bychom to na začátku trefili dokonale :laughing: Protože něco se třeba nestihne, takže očekávám, že to budeme upravovat během toho běhu.

Ale jako to jsou detaily, klidně to nechme takto.

Reply:
> že to budeme upravovat během toho běhu.

Urcite budeme ;)